README.rst

```diff
@@ -83,6 +82,62 @@ After running `aws-mfa`, your credentials file would read:
     aws_secret_access_key = <POPULATED_BY_AWS-MFA>
     aws_security_token = <POPULATED_BY_AWS-MFA>
 
+The default naming convention for the credential section can be overriden by using the ``--long-term-suffix`` and
+``--short-term-suffix`` command line arguments. For example, in a multi account scenario you can have one AWS account
```

Comment:
@broamski , sorry for the delayed response.  Internally we use aws-cli, boto3 and nodejs based tooling to manage multiple AWS accounts for multiple customers and multiple stages per customer. Because we have a lot of accounts it is mandatory that devops specify the AWS profile to use on the command line to avoid deploying changes to the wrong AWS account. I have added a simplified example at the end of the README file. 

Thanks for your idea of a script for managing short and long term credentials with AWS profiles. With the changes in this pull request it it just what we were looking for.